joomla/joomla-cms
# UX Editor button to simplify the insertion of a module on an article

#### What is this?

Another button bellow the editor that allows people to easily insert a module to their article
Once the button get their position inside tinyMCE this will look a lot better.
#### How to test

Apply this PR with patch tester
Go to discover and install the new plugin
Edit an article (fronted/backend) and make sure that the module is inserted
Also insert a style in the given input, either html5 or none, and observe the changes
#### Preview

![screen shot 2015-10-11 at 11 35 32](http://issues.joomla.org/uploads/1/8bb0fda942868e369aa370588c9ca848.png)
